### ISSUE TITLE
Create .lando folder for Lando scripts & config

### ISSUE THREAD

**Issue by tormi** [wunderio/drupal-project#61]
## Possible project structure

```
./
|-- .lando
   |-- .env
   |-- php.ini
   |-- syncdb.sh
|-- .lando.yml
```

## .lando.yml

```
name: project
recipe: drupal8

env_file:
  - .lando/.env

services:
  appserver:
    type: php
    config:
      php: .lando/php.ini

tooling:
  syncdb:
    description: Synchronise local database with stage environment
    cmd:
      - appserver: /app/.lando/syncdb.sh
```

## syncdb.sh

```
#!/bin/sh
set -exu

# Synchronise local database with stage environment
cd /app
drush @stage sql-dump --structure-tables-list=cache,cache_*,history,sessions,watchdog > /app/dump.sql
drush @local sql-drop -y
drush @local sql-query --file=/app/dump.sql -y
drush @local cc drush
drush @local cr
rm /app/dump.sql
```

**Comment by tormi** [comment]
cc: @floretan, @hkirsman

**Comment by tormi** [comment]
Resolved in #62.
